client/src/components/overview/ImgGallery/Large.jsx

```diff
@@ -31,13 +31,13 @@ const DefaultView = styled.div`
   background-position: center center;
 `;
 const RightArrow = styled.p`
-  font-size: 40px;
-  color: rgba(72, 72, 72, 0.7);
+  font-size: 80px;
+  color: ${(props) => (props.rightArrow ? 'rgba(72, 72, 72, 0.7)' : 'rgba(72, 72, 72, 0.0)')};
   padding-left: 50%;
 `;
 const LeftArrow = styled.p`
```

Comment:
shouldn't we use "styled.button" since the arrows are buttons?

Reply:
I could, but for styling purposes, I think the arrow text looks better than if I made it a button. The button would have an outline and background color for the element, but I just want the arrow text without any outline or background color. 

This is what it looks like if I made it a button:
![image](https://user-images.githubusercontent.com/57816663/121571225-94691b80-c9d7-11eb-807b-5a03d097c5ae.png)
But I want it like this:
![image](https://user-images.githubusercontent.com/57816663/121571260-a34fce00-c9d7-11eb-8fb2-22a22a3b1079.png)

Reply:
ah I agree it does look better like that!